plugins/ks.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ks
+spec:
+  version: v0.0.9
+  homepage: https://github.com/linuxsuren/ks
+  shortDescription: Make it be easy to manupulate Kubesphere
+  description: |
+    Kubesphere is enterprise-grade container platform tailored for multicloud and multi-cluster management
```

Comment:
This description might be helpful to people who already know kubesphere, but you should try to target a wider audience here.
Basically this description is a summary from https://github.com/kubesphere/kubesphere, but this plugin does not install kubesphere itself.

Please try to explain what your plugin does.

Reply:
Good point.